src/main/kotlin/com/hedvig/underwriter/web/dtos/InsuranceCompanyDto.kt

```diff
@@ -0,0 +1,11 @@
+package com.hedvig.underwriter.web.dtos
+
+import com.hedvig.underwriter.model.InsuranceCompany
+
+data class InsuranceCompanyDto(
+    val id: String,
+    val displayName: String,
+    val switchable: Boolean
+)
+
+fun InsuranceCompany.toDto(): InsuranceCompanyDto = InsuranceCompanyDto(id, displayName, switchable)
```

Comment:
Since they are in the same file, I usually see the function being inside the block
```suggestion
data class InsuranceCompanyDto(
    val id: String,
    val displayName: String,
    val switchable: Boolean
) {
    fun toDto() = InsuranceCompanyDto(id, displayName, switchable)
}
```

Reply:
They are not the same class. It a function on another class, `InsuranceCompany`, to convert to to this class, `InsuranceCompanyDto`.

Reply:
Aha! Sorry I'm sleeping 😴 Then this is nice!